src/js/services/forms.js

```diff
@@ -210,7 +210,7 @@ export default App.extend({
 
     const isReport = form.isReport();
 
-    if (isReport) opts.created = `<=${ this.action.get('created_at') }`;
+    if (isReport) opts.submitted = `<=${ this.action.get('submitted_at') }`;
```

Comment:
@nmajor25 , @paulfalgout , this is a bug. I think we still need to send the `created_at` timestamp, since this is the time _when the action was created_. I don't think `submitted_at` exists on the action, only on the form response.